src/uct/ib/base/ib_iface.c

```diff
@@ -1839,3 +1873,10 @@ ucs_status_t uct_ib_iface_arm_cq(uct_ib_iface_t *iface,
     }
     return UCS_OK;
 }
+
+uint8_t uct_ib_iface_get_ep_sl(const uct_ib_iface_t *iface, const uct_ep_params_t *params)
+{
+    return (params->field_mask & UCT_EP_PARAM_FIELD_PRIORITY) ?
+                   iface->config.priority_sls[params->priority] :
```

Comment:
so value of 0 in iface->config.priority_sls[params->priority means use default?
how can we use SL==0 then?

Reply:
Will change to `auto`